Gems/Pointcloud/Code/Source/Render/PointcloudFeatureProcessor.cpp

```diff
@@ -111,6 +159,10 @@ namespace Pointcloud
         {
             UpdateDrawPacket();
         }
+        if (auto it = m_pointcloudAssets.find(asset.GetId()); it != m_pointcloudAssets.end())
+        {
+            UpdatePointCloud(it->second.first, it->second.second->m_data, 0);
```

Comment:
I am no big fan of second.first.third it is spaghetti code.

Reply:
resolved